docker-compose.yaml

```
@@ -2,7 +2,7 @@ version: '3.9'
 
 services:
   postgres:
-    image: postgres:13.6
+    image: postgres:13.6-alpine
     container_name: db_helios
```

```suggestion
    container_name: helios_db
```
Je te propose de préxifer plutôt par Helios car quand on liste, c'est plus facile à retrouver quand on a en a plusieurs.